### PR TITLE
Link update (breaking change)

### PR DESCRIPTION
**Pull request by xelaint**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/graycoreio/daffodil/blob/master/CONTRIBUTING.md#commit
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?
No `daff-link-set` to use for sidebars.

Addresses breaking change in #427 

Fixes: #429 
Touches: #277

## What is the new behavior?
Update `daff-link` styles and create `daff-link-set` component.

## Does this PR introduce a breaking change?
```
[x] Yes
[ ] No
```

## Other information

### CODE REVIEW

File: apps/daffio/src/app/core/header/component/header.component.scss [hunk 1]
```diff
@@ -18,8 +18,10 @@
   display: block;
 
   &__item {
-    padding: 0 30px;
+    color: $black;
```
Comment (damienwebdev) [inline apps/daffio/src/app/core/header/component/header.component.scss]: This should be `currentColor` right?

Comment (xelaint) [reply apps/daffio/src/app/core/header/component/header.component.scss]: updated in c65aeb7

Comment (xelaint) [reply apps/daffio/src/app/core/header/component/header.component.scss]: updated in c65aeb7

**Comment by xelaint** [comment]
@damienwebdev commits updated.

**Pull request merged**